### ISSUE TITLE
(bug) /* inside a string makes the extension think brackets don't match

### ISSUE THREAD

**Issue by Barbiero** [sgwozdz/jenkinsfile-support#38]
minimal example:
```
pipeline {
//      ^~ Brackets don't match
    stages {
//        ^~ Brackets don't match
        stage("blah") {
//                   ^~ Brackets don't match
            steps {
//               ^~ Brackets don't match
                sh "mv folder/*.war somewhere.war"
            }
        }
    }
}
```

I believe the extension is misinterpreting the `/*` inside the string as the start of a comment.

**Comment by bartoncasey** [comment]
'//' has the same issue.

**Comment by sgwozdz** [comment]
thank you for sharing @Barbiero 
I will try to take a look in next few days

**Comment by riendeau** [comment]
Can work around this by ending the "comment" yourself. I changed
```groovy
junit '**/surefire-reports/*.xml'
```

to
```groovy
junit '**/surefire-reports/*.xml' // */
```

...and got rid of the errors.

**Comment by sgwozdz** [comment]
Merged to master, will be released with 1.1.0.
